src/util.ts

```diff
@@ -179,7 +179,7 @@ export function request(options: any) : Promise<any> {
  * console.log('cETH Address: ', Compound.util.getAddress(Compound.cETH));
  * ```
  */
-export function getAddress(contract: string, network='mainnet') : string {
+export function getAddress(contract: string, network='rsk-testnet') : string {
```

Comment:
```suggestion
export function getAddress(contract: string, network='testnet') : string {
```
Use same name as in constants